install-anp.yaml

```diff
@@ -80,7 +80,7 @@ spec:
       - name: kube-network-policies
         image: registry.k8s.io/networking/kube-network-policies:v0.3.0
         command: ["/bin/netpol"]
-        args: ["-v=2", "-admin-network-policy=true", "-baseline-admin-network-policy=true"]
+        args: ["-v=2", "-admin-network-policy=true", "-baseline-admin-network-policy=true","--hostname-override=${MY_NODE_NAME}"]
```

Comment:
the main problem because I added the env variables is because I was not sure you can do this, of using the env variable as the flag value and I was too lazy ... now that you mentioned I can see this is exactly how most people do https://grep.app/search?q=hostname-override%3D%24&filter[lang][0]=YAML

Reply:
ok, embarrassing things https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#use-environment-variables-to-define-arguments